src/style.css

```
@@ -0,0 +1,35 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  height: 15px;
+}
+
+.stars__item {
+  display: inline-block;
+  margin: 0 1px;
+  height: 12px;
+  width: 13px;
+  content: url("images/star.svg");
+}
+
+.stars--1 .stars__item:nth-child(1) {
```

```suggestion
.stars--1 .stars__item:first-child {
```
`first-child` is easier to understand than `nth-child(1)`